sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -39,7 +39,7 @@ You can create the resource using
 Below is an example of how you can create a Language resource using the CLI:
 
 ```bash
-# Create a new resource group to hold the text analytics resource -
+# Create a new resource group to hold the language resource -
 # if using an existing resource group, skip this step
 az group create --name my-resource-group --location westus2
 ```
```

Comment:
like 48 needs to be updated too. and will the `--kind` option should change? I haven't tried it

Reply:
It's still "TextAnalytics". Even in the Portal when selecting other language features, the kind used is TextAnalytics.